assets/src/api.ts

```diff
@@ -268,3 +275,19 @@ export const authorizeSlackIntegration = async (
     .set('Authorization', token)
     .then((res) => res.body.data);
 };
+
+export const createOrUpdateWidgetConfig = async (
+  widgetConfigParams: WidgetConfigParams,
+  id?: string,
+  token = getAccessToken()
+) => {
+  if (!token) {
+    throw new Error('Invalid token!');
+  }
+
+  return request
+    .put(`/api/widget_configs/${id}`)
```

Comment:
hmm i wonder if `widget_settings` would be a bit more "human readable"? (might just be me, but "configs" sounds kinda weird)

Reply:
i also don't think we need to pass in the widget_config `id`, if there's only one per account we should just do a lookup by `account_id` based on the authorized user -- what do you think?

Reply:
also a little confused after looking at the backend code below -- shouldn't this be calling the `create_or_update` endpoint on this line? https://github.com/papercups-io/papercups/pull/37/files#diff-d7c965d3c254345cb15f54194253be75R49